Testing-jobs-and-other-features-on-production.md

```diff
@@ -6,9 +6,9 @@ If your job or feature is not essential for the release and has not been fully t
 - The job or feature should run without failures on the Oppia backup server.
 - The job or feature produces the expected output.
 - The job or feature has the expected outcome (this must be verified by e.g. user-facing changes, or introspection of the datastore, or a validation job, or an output check, etc.).
-- The job or feature should be explicitly approved by server jobs admin (currently @seanlip and @vojtechjelinek).
+- The job or feature should be explicitly approved by server jobs admin (currently @seanlip,  @vojtechjelinek, @kevintab95 and @DubeySandeep).
 
-Also, in case your job is a new migration job (not any already existing migrations), there has to be an audit job accompanying it to verify that the data that you are migrating is valid in the server. **The audit job will have to be "Fully tested" before testing on the migration job can start.** 
+Also, in case your job is a new migration job (not any already existing migrations), there has to be an audit job accompanying it to verify that the data that you are migrating are valid in the server. **The audit job will have to be "Fully tested" before testing on the migration job can start.** 
```

Comment:
Keep it as "is" -- data is actually plural. (The singular is datum.)

Also change "any" to "an" in the parenthetical.